administration/src/cards/extensions/BirthdayExtension.tsx

```diff
@@ -24,20 +31,24 @@ const BirthdayForm = ({ value, setValue, isValid }: ExtensionComponentProps<Birt
     return null
   }
 
-  const changeBirthday = (date: Date | null) =>
+  const changeBirthday = (date: Date | null) => {
     setValue({ birthday: PlainDate.safeFromCustomFormat(date?.toLocaleDateString() ?? null) })
+    if (!touched) {
+      setTouched(true)
+    }
```

Comment:
```suggestion
    setTouched(true)
```
Same for other occurrences.